src/modules/kvs-watch/kvs-watch.c

```diff
@@ -703,43 +703,44 @@ static void watcher_respond_ns (struct ns_monitor *nsm)
         flux_log_error (nsm->ctx->h, "%s: zlist_dup", __FUNCTION__);
 }
 
```

Comment:
In commit message for a3d9e2355, second "to" should be "do"?

> Problem: kvs-watch is open to guest users, but the cancellations
and disconnects of the watches to not verify that the request came from the
same guest user